source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,161 @@
+  - `DJTechTools
+    review <https://djtechtools.com/2019/10/16/traktor-kontrol-s3-the-middle-child-for-the-mk3-traktor-generation/>`__
+  - `DJWORX
+    overview <https://djworx.com/the-traktor-kontrol-s3-we-have-it-but-not-the-software/>`__
+
+The Kontrol S3 is an introductory 4 deck controller with good build
+quality and integrated sound card. This is the first controller released
+with the "S3" name.
+
+The Kontrol S3 can run from USB bus power. Using the separate power
+supply increases the brightness of the LEDs, which is helpful for using
+it in daylight, and increases the volume of the headphone output.
+
+Compatibility
+-------------
+
+Controller
+~~~~~~~~~~
+
+The Kontrol S3 is a USB class compliant audio and HID device,
+so it is compatible with Mixxx without any proprietary drivers on
+GNU/Linux and Mac OS X. On Windows, it is recommended to install the
+`driver from Native
+Instruments <https://www.native-instruments.com/en/support/downloads/drivers-other-files/>`__
+and select the ASIO sound API in the Sound Hardware section of Mixxx's
+Preferences.
+
+With the S3 plugged in, a MIDI device is listed as an available
+controller in Mixxx's Preferences. The controller uses
+HID for the knobs, buttons, and other components on the device, so the
+mapping can only be loaded when you select the HID device on the left
+side of Mixxx's Preferences.
+
+Mapping description
+-------------------
+
+Note that Mixxx doesn't have the concept of a single "master" deck for
+sync. Instead, push and hold the sync button to "lock" sync on for all
+decks you want to remain in sync. Or you can push Shift + Sync to lock
+sync on. Refer to `the Mixxx
+manual <http://www.mixxx.org/manual/2.0/chapters/djing_with_mixxx.html#master-sync>`__
+for details.
+
+Mixer
+~~~~~
+
+  - Gain, equalizer high/mid/low, and cue (headphones)
+    behave as labelled.
+  - FX Enable buttons: See Effect section below.
+  - Filter: controls QuickEffect superknob. This controls the Filter
+    effect by default, but a different effect can be chosen in the
+    Equalizer section of Mixxs's Preferences.
+  - EXT: The EXT button changes the fourth channel pregain (knob), pfl, and volume (slider) adjustments to operate with the Microphone input.  The microphone does not respond to EQ or effects.
+  - Shift + EXT: Switches input from Mic to Line and back again.
+
+The Master Volume knob on the S3 controls the volume of the S3's master
+output in hardware, so it does not affect the software master gain knob
+in Mixxx by default. You can hold shift and turn the knob to adjust Mixxx's
+master gain.  Note that this will still also adjust the hardware gain, so
+after you release shift you'll need to adjust the knob again.
+
+Peak display is only generated from software. So if
+you see clipping indicated, lower the gain of the playing decks or use Shift + Master Volume.
+
+Decks
+~~~~~
+
+  - Pressing the library knob: Load track selected in library to the deck.
```

Comment:
I think this would benefit from tables, like  on the DDJ-200 page.

Reply:
done